tracer/src/Datadog.Trace/Generated/net461/Datadog.Trace.SourceGenerators/InstrumentationDefinitionsGenerator/InstrumentationDefinitions.g.cs

```diff
@@ -15,671 +15,8 @@ namespace Datadog.Trace.ClrProfiler
 {
     internal static partial class InstrumentationDefinitions
     {
-        internal static NativeCallTargetDefinition2[] Instrumentations;
-
         static InstrumentationDefinitions()
```

Comment:
I don't think we need to generate this file at all any more?

Reply:
`InterationOption` is still invoking the `GetIntegrationId` function...